core/input/input_event.cpp

```diff
@@ -796,7 +864,26 @@ bool InputEventJoypadMotion::action_match(const Ref<InputEvent> &p_event, bool *
 	return match;
 }
 
+static const char *_joy_axis_descriptions[JOY_AXIS_MAX] = {
+	TTRC("Left Stick X-Axis, Joystick 0 X-Axis"),
+	TTRC("Left Stick Y-Axis, Joystick 0 Y-Axis"),
+	TTRC("Right Stick X-Axis, Joystick 1 X-Axis"),
+	TTRC("Right Stick X-Axis, Joystick 1 Y-Axis"),
+	TTRC("Joystick 2 X-Axis, Left Trigger, Sony L2, Xbox LT"),
```

Comment:
Copy and paste error: The fourth entry should be "TTRC("Right Stick **Y**-Axis, Joystick 1 Y-Axis"),"